src/index.ts

```diff
@@ -94,6 +143,14 @@ interface Route {
 	try(path: string): string | undefined
 }
 
+interface Resp extends ServerResponse {
+	locals: {[key: string]: any}
+	log: () => this
+	typeLen: (type: string, len: number) => this
+}
+
+type ReqListener = (req: IncomingMessage, res: Resp) => void
```

Comment:
Same problem with naming.
```suggestion
type RequestListener = (req: IncomingMessage, res: Resp) => void
```